Makefile

```diff
@@ -39,10 +37,11 @@ container-codegen:
 	docker rm temp-codegen
 
 generate-fakes:
-	go generate ./...
+	$(GO) generate ./...
 
 clean:
 	@rm -rf ./bin
 
-opm-test:
-	 $(shell ./opm-test.sh || echo "opm-test FAIL")
+.PHONY: e2e
```

Comment:
super-nit: can we either put all phony directives together, or make them all separate like this?